data/json/monsters/zed_misc.json

```diff
@@ -1644,5 +1644,95 @@
       "FILTHY"
     ],
     "armor": { "bash": 5, "cut": 5, "bullet": 1, "electric": 1 }
+  },
+  {
+    "id": "mon_zombie_gecko",
+    "type": "MONSTER",
+    "name": { "str": "zombie gecko" },
+    "description": "The wound covered skin of this large gecko appears to be peeling in places, revealing a black goop beneath the surface. run away while you can.",
+    "default_faction": "zombie",
+    "species": [ "ZOMBIE", "REPTILE" ],
+    "volume": "120 L",
+    "weight": "150 kg",
+    "hp": 160,
+    "speed": 80,
+    "attack_cost": 50,
+    "material": [ "flesh" ],
+    "symbol": "G",
+    "color": "black_green",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "bash", "amount": 5 } ],
+    "dodge": 2,
+    "families": [ "prof_intro_biology", "prof_wp_zombie" ],
+    "bleed_rate": 0,
+    "vision_night": 7,
+    "harvest": "zombie",
+    "special_attacks": [ [ "GRAB_DRAG", 3 ] ],
+    "regen_morale": true,
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "KEENNOSE",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON"
+    ],
+    "upgrades": { "half_life": 34, "into": "mon_frog_mother" },
+    "armor": { "bash": 8, "cut": 2, "bullet": 1, "electric": 2 }
+  },
+  {
+    "id": "mon_evoluted_z_gecko",
+    "type": "MONSTER",
+    "name": { "str": "evoluted zombie gecko" },
+    "description": "a giant gecko with its mouth open in an ominous way and with its lifeless eyes stares into void, pray.",
```

Comment:
The descriptions in general do a lot of prescribing ("pray, run, despair") instead of describing. I'd go lighter on the former.